exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/OtelScopeMode.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.prometheus;
+
+public enum OtelScopeMode {
+  /** No scope information is exported. */
+  DISABLED,
```

Comment:
Long term should aim to get rid of this? Does the spec recommend / encourage an option like this? If not, it just seems like it needlessly exposes users to metric identity conflicts.